automation/e2e-source.sh

```diff
@@ -3,3 +3,6 @@
 export DEV_MODE="${DEV_MODE:-false}"
 export STORAGE_CLASS="${STORAGE_CLASS:-}"
 export NUM_NODES=${NUM_NODES:-2}
+export REGISTRY_IMAGE_DESTINATION="quay.io/kubevirt/e2e-tests-example-vm"
```

Comment:
Is there absolutely no way to use an ephemeral registry? Why do we have to store test artifacts on a live registry? (quay.io)

Reply:
I replied [here](https://github.com/kubevirt/kubevirt-tekton-tasks/pull/549#discussion_r1839821263) earlier. The issue is that there is no built-in registry in Kubernetes cluster (only on OpenShift).

So we have 3 options:

1). Deploy a new pod that will use [registry image](https://hub.docker.com/_/registry) to push there
2). Use OCP's built-in container registry
3). Use an external container registry

@0xFelix @ksimon1 What do you prefer?